articles/140_topic_and_service_name_mapping.md

```diff
@@ -243,41 +252,71 @@ Here is a non-exhaustive list of prefixes:
+Please bare in mind, that the length of the partition gets further diminished due to the introduction of a ros specific prefix.
+The actual length of a ROS Topic, including the namespace hierarchy and the base name of the topic, may thus be varying in length as well.
+Yet, the base name token must not exceed the length of 256 characters as this is getting mapped direclty as the DDS topic.
+
 ### Communicating with Non-ROS Topics
 
 Since all ROS topics are prefixed when being converted to DDS topic names, it makes it impossible to subscribe to existing DDS topics which do not follow the same naming pattern.
-For example, if an existing DDS program is publishing on the `image` topic (and is using the DDS equivalent to the ROS message type) then a ROS program could not subscribe to it because of the name mangling.
+For example, if an existing DDS program is publishing on the `image` topic (and is using the DDS equivalent to the ROS message type) then a ROS program could not subscribe to it because of the name mangling produced by the implicit ROS specific namespace.
 Therefore to allow ROS programs to interoperate with "native" DDS topic names the API should provide a way to skip the ROS specific prefixing.
 
-### ROS to DDS Name Conversion Examples
+#### Examples and Ideas for Communicating with Non-ROS Topics
 
-Here are some examples of translations between ROS topic names and the corresponding DDS topic names:
+Note that these are not part of the proposal, but only possible solutions to the issue of communicating with "native" DDS topics.
+This section should be update when/if a complete solution is found.
 
-| ROS Name          | Subsystem | DDS Name               |
-|-------------------|-----------|------------------------|
-| `/foo`            | Topic     | `rt__foo`              |
-| `/foo/bar`        | Topic     | `rt__foo__bar`         |
-| `/_foo/bar`       | Topic     | `rt___foo__bar`        |
-| `/foo/_bar`       | Topic     | `rt__foo___bar`        |
-| `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
-| `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
+It may be possible, through an option in the API or through some name syntax, to get a similar break down but without the ROS specific prefix.
 
-### ROS Topic and Service Name Length Limit
+For example with an option in the API:
 
-Because DDS topic names must be limited to 255 characters, the length of a ROS topic or service name is also limited in length.
-In the case of a topic, the length is governed by the following algorithm:
+| ROS Name              | ROS Prefix | DDS Topic   | DDS Partition               |
+|-----------------------|------------|-------------|-----------------------------|
+| `rostopic://image`    | with       | `image`     | [`rt`]                      |
+| `rostopic://image`    | without    | `image`     | [``]                        |
```

Comment:
Oh, I see. These are actual backticks which are only used for formatting. Not actually quotes. Thanks for clarifying.